debezium-core/src/test/java/io/debezium/transforms/outbox/EventRouterTest.java

```diff
@@ -911,7 +911,7 @@ public void canExpandJsonPayloadIfConfigured() {
                 "UserCreated",
                 "10711fa5",
                 "User",
-                "{\"fullName\": \"John Doe\", \"enabled\": true, \"rating\": 4.9, \"age\": 42, \"pets\": [\"dog\", \"cat\"]}",
+                "{\"fullName\": \"John Doe\", \"enabled\": true, \"rating\": 4.9, \"age\": 42, \"pets\": [{\"type\": \"dog\"}, {\"type\": \"cat\"}]}",
```

Comment:
Perhaps it would be nicer to create a dedicated test for an array of objects instead of changing the existing test. I'm happy to apply the changes if needed

Reply:
That typically depends on how you change an existing test.  For example, if you had introduced a new field here with the object array that would have been fine; however if you change an existing field then we end up potentially losing some test coverage for another variant of a data type.  Would it be possible to add a new field here for your use case, i.e.

```
"\pets\": [\"dog\", \"cat\"], \"pets_objarray\": [{\"type\": \"dog\"}, {\"type\": \"cat\"}]
```